dashboard/src/index.scss

```diff
@@ -3,10 +3,32 @@
 @import "~@cds/core/styles/theme.dark.css"; // clarity core dark theme
 @import "~@cds/core/styles/module.shims.css"; // non-evergreen browser shims
 @import "~@cds/city/css/bundles/default.css"; // load base font
-// clarity UI light/dark themes should be conditionalled loaded since they don't support dynamic theming
+// clarity UI light/dark themes are loaded by changing <link href=...>, since they aren't prepared for theming
 
 svg.icon {
   /* Fix positioning */
   position: relative;
   bottom: -0.125em;
 }
+
+// CSS tweaks the dark mode theme theme
+[cds-theme~="dark"] {
+  %convert-to-white {
+    filter: invert(1) brightness(100%) saturate(0%);
+  }
+
+  .section-not-found img {
+    // White 404 logo
+    @extend %convert-to-white;
+  }
+
+  .kubeapps-header-subtitle img {
+    // White header subtitle Helm logo
+    @extend %convert-to-white;
+  }
+
+  .bg-img img {
+    // White Helm logo in card overlay
+    @extend %convert-to-white;
+  }
```

Comment:
:+1: 

I wonder if we will miss whenever we add new imgs

Reply:
Probably, but, since we have the toggle button, we can test it in runtime :p 
However, what concerns me the most, is branding compliance; changing colors arbitrarily is usually not supported by brand guidelines. However, none of our current icons is a trademark, but we should be aware of that in the future,